perf/conf/testsuite/Iterate.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+  <property>
+    <name>iterations</name>
+    <value>2</value>
+    <description>the number of the write-read iteration</description>
+  </property>
+  <property>
+    <name>shuffle.mode</name>
+    <value>false</value>
+    <description>shuffle mode means it may read remotely</description>
```

Comment:
`workers will only read the local dataset by default, set shuffle mode to true to read from the entire dataset.`